mesh/meshdb.go

```diff
@@ -71,7 +71,8 @@ func (m *MeshDB) Close() {
 
 func (m *MeshDB) AddBlock(bl *types.Block) error {
 	if _, err := m.getMiniBlockBytes(bl.ID()); err == nil {
-		return errors.New(fmt.Sprintf("block %v already exists in database", bl.ID()))
+		log.Warning(fmt.Sprintf("block %v already exists in database", bl.ID()))
```

Comment:
change to error

Reply:
done, though I find it weird to print an error but return from the function as there was no error